### ISSUE TITLE
[Docs] New tutorial/analysis section

### ISSUE THREAD

**Issue by jeromekelleher** [tskit-dev/msprime#1001]
This section is to help users understand how to effectively run lots of simulations and how to analyse then.

Can start by moving the current ``advanced features/replication`` section in there.

I think @gtsambos and @agladstein have some ideas for a section in there on running simulations from priors?

**Comment by agladstein** [comment]
Yes, https://github.com/tskit-dev/msprime/issues/825 would be part of that.

**Comment by jeromekelleher** [comment]
This sounds like a topic much more suited to the "tskit tutorials" site. See proposal here: tskit-dev/tutorials#10

Shall we migrate this issue to the "tutorials" repo?

**Comment by agladstein** [comment]
> Shall we migrate this issue to the "tutorials" repo?

Yes!

**Comment by jeromekelleher** [comment]
OK, will do once we have buy-in for a few more people on the tutorials repo side.
